src/adapters/rhythmone.js

```diff
@@ -224,53 +186,76 @@ module.exports = function(bidManager, global, loader){
     return endpoint;
   }
   
+  function sendAuditBeacon(placementId){
+    var data = {
+        doc_version : 1,
+        doc_type : "Prebid Audit",
+        placement_id: placementId
+      },
+      ao = document.location.ancestorOrigins,
+      q = [],
+      u = "//hbevents.1rx.io/audit?",
+      i = new Image();
+    
+    if(ao && ao.length > 0){
+      data.ancestor_origins = ao[ao.length-1];
+	}
+    
+    data.popped = window.opener!==null?1:0;
+    data.framed = window.top===window?0:1;
```

Comment:
With all due respect, this is unnecessary.  A security exception is only thrown when accessing the properties of top.  The window object itself can be compared safely.